html/openstef_dashboard_doc.html

```diff
@@ -0,0 +1,53 @@
+<!--
+SPDX-FileCopyrightText: 2017-2021 Contributors to the OpenSTF project <korte.termijn.prognoses@alliander.com>
+
+SPDX-License-Identifier: MPL-2.0
+-->
+<!DOCTYPE html>
+<html class="writer-html5" lang="en" >
+
+<head>
+    <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<title>Description of OpenSTEF dashboard</title>
+<script src="https://code.jquery.com/jquery-1.7.1.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/imagemapster/1.5.4/jquery.imagemapster.min.js"></script>
+<style type="text/css">
+/* set up the font to be used for the page */
+body {font-family: tahoma; arial, sans-serif;}
+
+/* add the style for the link span text - first line */
+#imap a span:first-line {font-weight:bold; font-style:italic;}
+</style>
+
+</head>
+
+<body>
+
+<img id="OPENSTEF" src="https://user-images.githubusercontent.com/60883372/146760483-29af3ac7-62af-4f13-98c7-982a79c517d1.jpg"
```

Comment:
Heb je een tool gebruikt om deze imagemap te maken? Zo ja, dan zou een kleine comment hier over welke dat is een volgende persoon helpen, mochten we nog een keer dingen wijzigen